Playground/RxSwiftExtPlayground.playground/Pages/UIViewProperty+Rx.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,85 @@
+/*:
+ > # IMPORTANT: To use `RxSwiftExtPlayground.playground`, please:
+
+ 1. Make sure you have [Carthage](https://github.com/Carthage/Carthage) installed
+ 1. Fetch Carthage dependencies from shell: `carthage bootstrap --platform ios`
+ 1. Build scheme `RxSwiftExtPlayground` scheme for a simulator target
+ 1. Choose `View > Show Debug Area`
+ */
+
+//: [Previous](@previous)
+
+import RxSwift
+import RxSwiftExt
+import PlaygroundSupport
+import UIKit
+
+/*:
+ ## animate
+
+ The `animate` operators provide a Completable that when subscribed to starts the animation (after a delay or not) and completes once the animation is ended
+ */
+
+class AnimateViewController : UIViewController {
+
+    lazy var box1: UIView = {
+        let view = UIView(frame: CGRect(x: 100, y: 100, width: 100, height: 100))
+        view.backgroundColor = .red
+        return view
+    }()
+
+    lazy var box2: UIView = {
+        let view = UIView(frame: CGRect(x: 100, y: 220, width: 100, height: 100))
+        view.backgroundColor = .green
+        return view
+    }()
+
+    lazy var box3: UIView = {
+        let view = UIView(frame: CGRect(x: 100, y: 340, width: 100, height: 100))
+        view.backgroundColor = .blue
+        return view
+    }()
+
+    lazy var animator1 = {
+        UIViewPropertyAnimator(duration: 1, curve: .linear) { [unowned self] in
+            self.box1.transform = CGAffineTransform(translationX: 0, y: -100)
+        }
+    }()
+
+    lazy var animator2 = {
+        UIViewPropertyAnimator(duration: 1, curve: .linear) { [unowned self] in
+            self.box2.transform = CGAffineTransform(translationX: 0, y: -100)
+        }
+    }()
+
+    lazy var animator3 = {
+        UIViewPropertyAnimator(duration: 1, curve: .linear) { [unowned self] in
+            self.box3.transform = CGAffineTransform(translationX: 0, y: -100)
+        }
+    }()
+
+    let disposeBag = DisposeBag()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // construct the main view
+        self.view.backgroundColor = .white
+        self.view.addSubview(self.box1)
+        self.view.addSubview(self.box2)
+        self.view.addSubview(self.box3)
+
+        // trigger the animation chain
+        self.animator1.rx.animate
```

Comment:
Two quick notes about this example: 

1. I'd love it if the animations could be different, different speeds or scale and translate etc
2. It would be great if we could have a button on screen to trigger the animation. That way its obvious how to trigger the animation from a user event
3. Can we add a note somewhere about the fact they should open the Assistant Editor to see the live view?

Reply:
done.